.github/workflows/ruby-build.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     steps:
       - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v3.3.0
       - name: Set up Ruby ${{ matrix.python-version }}
-        uses: ruby/setup-ruby@250fcd6a742febb1123a77a841497ccaa8b9e939 # v4.5.0
+        uses: ruby/setup-ruby@5311f05890856149502132d25c4a24985a00d426 # v4.5.0
```

Comment:
NB: Dependabot doesn't appear to be updating the version comments here.

Reply:
If it finds the current version differs, it doesn’t update it for some reason. Know anyone who knows the dependabot codebase?

Reply:
Unfortunately not 😅 -- I can file an upstream issue, and manually fixup here for the time being.

Reply:
Looks like someone has already filed this: https://github.com/dependabot/dependabot-core/issues/7912